examples/models/qwen3/qwen3_32b_decode_fused_qkrope.py

```
@@ -0,0 +1,781 @@
+                        ctx_flat_bf16 = pl.cast(ctx_flat, target_type=pl.BF16)
+                        attn_row = pl.assemble(
+                            attn_row, ctx_flat_bf16, [0, q_base * head_dim],
+                        )
+
+                attn_out = pl.assemble(attn_out, attn_row, [b, 0])
+
+            # ── Scope 3: output projection + residual + post RMSNorm + MLP + residual ──
+            for b0 in pl.range(0, batch, BATCH_TILE):
+                resid1_tile = pl.create_tensor([BATCH_TILE, hidden], dtype=pl.FP32)
+
+                # Stage 1: Output projection: attn_out × wo, tiled by Q_OUT_CHUNK.
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)